### PR TITLE
remove userlist component from jazzbar template

### PR DESCRIPTION
**Pull request by denisdimitrov**
Removes the UserList component from the Jazzbar template only.

Fixes: https://github.com/sparkletown/internal-sparkle-issues/issues/235

### CODE REVIEW

**Review by 0xdevalias — APPROVED** [review]
Tentatively approved, pending a satisfactory answer to the question raised in https://github.com/sparkletown/internal-sparkle-issues/issues/235#issuecomment-768685324

> Reason for this change: Essentially it's superfluous information, since you can already see who is there based on who is sitting at the tables/etc.
>
> Though this does raise a question for me.. what happens if a user is in the jazz bar but not sitting at a table? Will they be shown anywhere then? And if not, how are we going to convey this information if we remove `UserList` from here?

**Comment by denisdimitrov** [comment]
@0xdevalias I left a comment in the issue as well. But since it's not an actual answer to the question you brought up, I suggest waiting for the daily before merging this.

**Comment by 0xdevalias** [comment]
https://github.com/sparkletown/internal-sparkle-issues/issues/235#issuecomment-769648581

> In our standup today, @sparkleed said that it's fine for us to not delete this for now, since we will be looking at redesigning this code soon anyway.

**Pull request closed without merging**
